src/client/utils/browser.js

```diff
@@ -32,16 +26,46 @@ export var isIE11 = /trident\/7.0/.test(userAgent) && !(browser.name === 'msie'
 if (isIE11)
     majorBrowserVersion = 11;
```

Comment:
why we need to store `majorBrowserVersion` and recalculate `browser` and `engine` version. We can do it one time (like `majorBrowserVersion`). `majorBrowserVersion` now isn't good name (see https://github.com/DevExpress/testcafe-hammerhead/pull/781/files#r78536076).